src/Perimeterx.php

```diff
@@ -151,8 +151,22 @@ private function handleVerification($pxCtx)
                 die();
             }
         } else {
-            $this->pxActivitiesClient->sendToPerimeterx('page_requested', $pxCtx, ['module_version' => $this->pxConfig['sdk_name'], 'http_version' => $pxCtx->getHttpVersion(), 'http_method' => $pxCtx->getHttpMethod()]);
+            $this->sendActivity('page_requested', $pxCtx, ['module_version' => $this->pxConfig['sdk_name'], 'http_version' => $pxCtx->getHttpVersion(), 'http_method' => $pxCtx->getHttpMethod()]);
             return 1;
         }
     }
+
+    /**
+     * @param string            $activityType
+     * @param PerimeterxContext $pxCtx
+     * @param array             $details
+     */
+    private function sendActivity($activityType, $pxCtx, $details = [])
+    {
+        if (isset($this->pxConfig['custom_activity_handler'])) {
+            $this->pxConfig['custom_activity_handler']($activityType, $pxCtx, $details);
```

Comment:
hey Paul, what do you think about executing our function too along with the custom handler? without that, we will not get blocking statistics back to our servers and therefor will be unable to display it in your data.